packages/amplication-server/src/core/outdatedVersionAlert/outdatedVersionAlert.service.ts

```diff
@@ -50,4 +52,48 @@ export class OutdatedVersionAlertService {
   ): Promise<OutdatedVersionAlert | null> {
     return this.prisma.outdatedVersionAlert.findUnique(args);
   }
+
+  async triggerAlertsForTemplateVersion(
+    templateResourceId: string,
+    outdatedVersion: string,
+    latestVersion: string
+  ) {
+    const template = await this.resourceService.resource({
+      where: {
+        id: templateResourceId,
```

Comment:
I would add to the `where` also a validation on the resource type to verify it's a template and not another type of resource